nexus/reconfigurator/planning/src/planner.rs

```diff
@@ -1263,13 +1356,9 @@ impl<'a> Planner<'a> {
         Ok(PlanningMgsUpdatesStepReport::new(pending_updates))
     }
 
-    /// Update at most one existing zone to use a new image source.
-    fn do_plan_zone_updates(
-        &mut self,
-        mgs_updates: &PlanningMgsUpdatesStepReport,
-    ) -> Result<PlanningZoneUpdatesStepReport, Error> {
-        let mut report = PlanningZoneUpdatesStepReport::new();
-
+    fn get_zones_not_yet_propagated_to_inventory(
```

Comment:
I'm confused by the phrase "zones not yet propagated to inventory".  Can we document that?